rollup.utils.js

```diff
@@ -120,7 +120,7 @@ function copyToPlaygrounds() {
             path.join(REPO_ROOT_DIR, "build"),
             playgroundDir
           );
-          await fse.copy(writtenDir, destDir);
+          await fse.copySync(writtenDir, destDir);
```

Comment:
No need for `await` anymore
```suggestion
          fse.copySync(writtenDir, destDir);
```

Reply:
Good call.